arm/Microsoft.SignalRService/webPubSub/deploy.bicep

```diff
@@ -0,0 +1,161 @@
+@description('Optional. The location to deploy the Web PubSub service.')
```

Comment:
```suggestion
@description('Optional. Location for all resources.')
```

Reply:
Note: Any updates of this file will eventually require to re-run the `Set-ModuleReadMe` script again